pkgs/development/compilers/halvm/bootloader.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
Please import dependencies separate here rather then importing `pkgs`. Same thing for the instances.